src/ipfsd-daemon.js

```diff
@@ -62,6 +62,21 @@ class Daemon {
     this._env = Object.assign({}, process.env, this.opts.env)
   }
 
+  /**
+   *
+   * @private
+   */
+  get runningNodeApi () {
```

Comment:
In addition to the `@private` notation, can we also underscore the name: `_runningNodeApi`

Reply:
I've removed the private annotation, there is no need for it to be private.